webapp/channels/src/packages/mattermost-redux/src/selectors/entities/teams.ts

```diff
@@ -145,6 +145,14 @@ export const getCurrentRelativeTeamUrl: (state: GlobalState) => string = createS
     },
 );
 
+export const getRelativeTeamUrl: (state: GlobalState, teamID: string) => string = createSelector(
```

Comment:
If we needed this to be memoized, it should be a selector factory since multiple callers would overwrite each other's memoized results.

That said, this doesn't seem like it needs to be memoized since it's just making a string each time, so I think you could remove the `createSelector` and just write it as a plain function that gets the team and returns the string

Reply:
Refactored to a simple function.